README.md

```diff
@@ -229,3 +229,9 @@ Dispatch the [Deploy Storybook](https://github.com/Constructor-io/constructorio-
 ## Usage examples
 - [Javascript](https://codesandbox.io/s/autocomplete-ui-integration-plain-y9zjl7)
 - [Remix App](https://codesandbox.io/p/sandbox/remix-example-for-constructorio-ui-autocomplete-kk5vh5?file=%2Fapp%2Froutes%2Findex.tsx)
+
+
+## License
+[MIT License](./LICENSE)
+
+Copyright (c) 2022-present Constructor
```

Comment:
I don't know if it matters but in the license our name is `Constructor.io Corporation`

Reply:
Yeah, I was thinking the same thing. Not sure if we should pick the preferred branding vs the legal name. I asked @sblaurock in our "Releasing an Open Source Library" Slab Template

Reply:
Thank you!